moto/ssm/models.py

```diff
@@ -14,10 +14,12 @@
 
 
 class Parameter(BaseModel):
-    def __init__(self, name, value, type, description, keyid, last_modified_date, version):
+    def __init__(self, name, value, type, description, allowed_pattern, keyid,
+            last_modified_date, version):
```

Comment:
I couldn't determine if there is a preferred line-wrapping style or line length for this project, so I tried to copy what already existed for 80 characters.